src/web/modules/gateway_accounts/worldpay_exemption_engine.njk

```diff
@@ -0,0 +1,49 @@
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% extends "layout/layout.njk" %}
+
+{% set isTestData = account and not (account.type === "live") %}
+
+{% block main %}
+  <h1 class="govuk-heading-m">Worldpay 3DS Exemption Engine</h1>
+
+  <div>
+    <a href="/gateway_accounts/{{ account.gateway_account_id }}" class="govuk-back-link">Gateway account ({{ account.gateway_account_id }})</a>
+  </div>
+
+  <p class="govuk-body">The Worldpay Exemption Engine uses real-time risk analysis to exempt as many transactions as possible from SCA (Strong Customer Authentication).</p>
+  <p class="govuk-body">This feature will be requested by a service through support.</p>
+
+  <form method="POST" action="/gateway_accounts/{{ account.gateway_account_id }}/worldpay_exemption_engine">
+
+    {{ govukRadios({
+      name: "enabled",
+      fieldset: {
+        legend: {
+          text: "Should the Exemption Engine be enabled?",
+          isPageHeading: false,
+          classes: "govuk-fieldset__legend--m"
+        }
+      },
+      items: [
+        {
+          value: "enabled",
+          text: "Enabled",
+          checked: enabled
+        },
+        {
+          value: "disabled",
+          text: "Disabled",
+          checked: not enabled
+        }
+      ]
+    }) }}
```

Comment:
This could just be two radio buttons labelled ‘Turn on Exemption Engine’ and ‘Turn off Exemption Engine’.

Reply:
Thanks for the suggestion. I've updated this and also updated the recurring payments and prepaid cards pages to be similar.
<img width="597" alt="Screenshot 2024-01-12 at 15 26 55" src="https://github.com/alphagov/pay-toolbox/assets/5648592/4e0cd43b-76ac-478c-a96c-3b71edd22f4a">

<img width="597" alt="Screenshot 2024-01-12 at 15 31 47" src="https://github.com/alphagov/pay-toolbox/assets/5648592/627f22d9-0d59-4b52-a947-536085db2540">

<img width="597" alt="Screenshot 2024-01-12 at 15 32 14" src="https://github.com/alphagov/pay-toolbox/assets/5648592/589b2aed-bc45-4f3c-9a7a-5de89cd3baa9">

Reply:
Updated the recurring payments settings page again to use "Turn on" and "Turn off"
<img width="597" alt="Screenshot 2024-01-12 at 15 36 42" src="https://github.com/alphagov/pay-toolbox/assets/5648592/5991fce8-f7d1-464d-9126-6fdd6cb49499">